### PR TITLE
Rasa version

### PR DESCRIPTION
**Pull request by nelealbers**
Changes Rasa version to 2.8 based on issue #55.

### CODE REVIEW

File: Rasa_Bot/.keras/keras.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "floatx": "float32",
+    "epsilon": 1e-07,
+    "backend": "tensorflow",
+    "image_data_format": "channels_last"
+}
```
Comment (svenvanderburg) [inline Rasa_Bot/.keras/keras.json:6]: Did you intend to commit this? Is this because in the back rasa uses keras?

Comment (svenvanderburg) [reply Rasa_Bot/.keras/keras.json:6]: And do we really need to specify the image_data_format? We don't use image data?

Comment (svenvanderburg) [reply Rasa_Bot/.keras/keras.json:6]: If this is all necessary for rasa and you also don't know why that's also fine and you can ignore these comments ;)

**Review by svenvanderburg — APPROVED** [review]
Looks good! Good to remove some older models as well!
I just added one minor comment.

File: Rasa_Bot/.keras/keras.json (hunk 1 above)
Comment (nelealbers) [reply Rasa_Bot/.keras/keras.json:6]: Honestly I also do not know what it is used for. It appeared when I installed the new rasa version.

Comment (nelealbers) [reply Rasa_Bot/.keras/keras.json:6]: @svenvanderburg I also just realized, I also need to update the requirements file outside of the Rasa_Bot-folder based on the new Rasa version right? I had not done that yet.

Comment (svenvanderburg) [reply Rasa_Bot/.keras/keras.json:6]: OK, good to check if it's necessary to commit it. You can also add it to `.gitignore`.

Comment (svenvanderburg) [reply Rasa_Bot/.keras/keras.json:6]: Yes, you need to update the root-folder requirements file!

Comment (nelealbers) [reply Rasa_Bot/.keras/keras.json]: Okay, I have updated the root-folder requirements and added .keras to gitignore

Comment (nelealbers) [reply Rasa_Bot/.keras/keras.json]: Ill merge then once the tests have completed.

**Pull request merged**